tests/e2e/e2e_cli_util.go

```diff
@@ -104,24 +105,23 @@ func queryGaiaDenomBalance(endpoint, addr, denom string) (sdk.Coin, error) {
 }
 
 func queryGovProposal(endpoint string, proposalID int) (govv1beta1.QueryProposalResponse, error) {
-	var emptyProp govv1beta1.QueryProposalResponse
+	var govProposalResp govv1beta1.QueryProposalResponse
 
 	path := fmt.Sprintf("%s/cosmos/gov/v1beta1/proposals/%d", endpoint, proposalID)
 
 	resp, err := http.Get(path) //nolint:gosec // this is only used during tests
 	if err != nil {
-		return emptyProp, fmt.Errorf("failed to execute HTTP request: %w", err)
+		return govProposalResp, fmt.Errorf("failed to execute HTTP request: %w", err)
```

Comment:
Would've preferred to keep the group stuff in a separate PR

Reply:
Yeah started to get ahead of myself, I'll follow up with a more thorough refactor pr